File: code/jvm/src/main/kotlin/pt/isel/leic/ptgest/domain/trainee/model/Trainee.kt
package pt.isel.leic.ptgest.domain.trainee.model

import pt.isel.leic.ptgest.domain.user.Gender
import java.util.*

data class Trainee(
    val traineeId: UUID,
    val traineeName: String,
    val gender: Gender,
    val trainerId: UUID?,
    val trainerName: String?
)

File: code/js/src/views/user/companiesViews/models/Trainee.ts
class Trainee {
    traineeId: string
    traineeName: string
    gender: string
    trainerId: string
    trainerName: string

    constructor(traineeId: string, traineeName: string, gender: string, trainerId: string, trainerName: string) {
        this.traineeId = traineeId
        this.traineeName = traineeName
        this.gender = gender
        this.trainerId = trainerId
        this.trainerName = trainerName
    }
}

export default Trainee
